alibi/explainers/pd_variance.py

```diff
@@ -0,0 +1,846 @@
+                     line_kw: Optional[dict] = None,
+                     fig_kw: Optional[dict] = None):
+    """
+    Plot feature importance and feature interaction based on partial dependence curves on matplotlib axes.
+
+    Parameters
+    ----------
+    exp
+        An `Explanation` object produced by a call to the
+        :py:meth:`alibi.explainers.pd_variance.PartialDependenceVariance.explain` method.
+    features
+        A list of features entries provided in `feature_names` argument  to the
+        :py:meth:`alibi.explainers.pd_variance.PartialDependenceVariance.explain` method, or
+        ``'all'`` to  plot all the explained features. For example, if  ``feature_names = ['temp', 'hum', 'windspeed']``
+        and we want to plot the values only for the ``'temp'`` and ``'windspeed'``, then we would set
+        ``features=[0, 2]``. Defaults to ``'all'``.
+    targets
+        A target name/index, or a list of target names/indices, for which to plot the feature importance/interaction,
+        or ``'all'``. Can be a mix of integers denoting target index or strings denoting entries in
+        `exp.meta['params']['target_names']`. By default ``'all'`` to plot the importance for all features
+        or to plot all the feature interactions.
+    summarise
+        Whether to plot only the summary of the feature importance/interaction as a bar plot, or plot comprehensive
+        exposition including partial dependence plots and conditional importance plots.
+    n_cols
+        Number of columns to organize the resulting plot into.
+    sort
+        Boolean flag whether to sort the values in descending order.
+    top_k
+        Number of top k values to be displayed if the ``sort=True``. If not provided, then all values will be displayed.
+    plot_limits
+        Minimum and maximum y-limits for all the line plots. If ``None`` will be automatically inferred.
+    ax
+        A `matplotlib` axes object or a `numpy` array of `matplotlib` axes to plot on.
+    sharey
+        A parameter specifying whether the y-axis of the PD and ICE curves should be on the same scale
+        for several features. Possible values are: ``'all'`` | ``'row'`` | ``None``.
+    bar_kw
+        Keyword arguments passed to the `matplotlib.pyplot.barh`_ function.
+    line_kw
+        Keyword arguments passed to the `matplotlib.pyplot.plot`_ function.
+    fig_kw
+        Keyword arguments passed to the `matplotlib.figure.set`_ function.
+
+        .. _matplotlib.pyplot.barh:
+            https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.barh.html
+
+        .. _matplotlib.pyplot.plot:
+            https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.plot.html
+
+        .. _matplotlib.figure.set:
+            https://matplotlib.org/stable/api/figure_api.html
+
+    Returns
+    -------
+    `plt.Axes` with the feature interaction plot.
+    """
+    # sanity check for `top_k`
+    if sorted and (top_k is not None) and (top_k <= 0):
```

Comment:
This is a bug since the parameter is `sort`, the reason it doesn't fail is because `sorted` is a built-in function... Plotting function tests would be useful!

Reply:
good catch